p2p/simulations/http.go

```diff
@@ -137,6 +141,13 @@ func (c *Client) SubscribeNetwork(events chan *Event, current bool) (event.Subsc
 	return event.NewSubscription(producer), nil
 }
 
+// SubscribeNetwork subscribes to network events which are sent from the server
+// as a server-sent-events stream, optionally receiving events for existing
+// nodes and connections
+func (c *Client) SubscribeNetwork(events chan *Event, current bool) (event.Subscription, error) {
```

Comment:
To avoid having both `SubscribeNetwork` and `SubscribeNetworkWithMsgFilter`, we could add an options struct, for example:

```go
type SubscribeOpts struct {
	Current bool
	Filter  string
}

func (c *Client) SubscribeNetwork(events chan *Event, opts SubscribeOpts) (event.Subscription, error) {
	
}
```